app/src/main/java/de/tum/in/tumcampusapp/models/tumcabe/WifiMeasurement.kt

```diff
@@ -1,9 +1,23 @@
 package de.tum.`in`.tumcampusapp.models.tumcabe
 
-data class WifiMeasurement(var date: String = "",
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+import de.tum.`in`.tumcampusapp.auxiliary.Utils
+import java.util.*
+
+@Entity(tableName = "wifi_measurement")
+data class WifiMeasurement(@PrimaryKey
+                           var date: String = "",
                            var ssid: String = "",
                            var bssid: String = "",
                            var dBm: Int = -1,
                            var accuracyInMeters: Float = -1f,
                            var latitude: Double = -1.0,
-                           var longitude: Double = -1.0)
+                           var longitude: Double = -1.0) {
+    companion object {
+        fun create(ssid: String, bssid: String, dBm: Int, accuracyInMeters: Float, latitude: Double, longitude: Double): WifiMeasurement {
+            val now = Utils.getDateTimeString(Date())
+            return WifiMeasurement(date = now, ssid = ssid, bssid = bssid, dBm = dBm, accuracyInMeters = accuracyInMeters, latitude = latitude, longitude = longitude)
```

Comment:
Why not directly do this in the declaration?
`var date: String = Utils.getDateTimeString(Date())`